src/components/history/History.js

```diff
@@ -100,7 +91,7 @@ export default class History extends PureComponent {
       return
     }
 
-    if (messages !== this.props.messages) {
+    if (!isEqual(messages, this.props.messages)) {
```

Comment:
This is the relevant part of the PR

Reply:
Is that everything what needs to be changed? 👍